KeyVaultClientEncryption/KeyVaultClientEncryption.cs

```diff
@@ -0,0 +1,240 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage;
+using Azure.Core;
+using Azure.Security.KeyVault.Keys.Cryptography;
+using Azure.Security.KeyVault.Keys;
+using Azure.Identity;
+using System;
+using System.Text;
+using System.IO;
+
+using System.Configuration;
+using System.Diagnostics;
+
+namespace KeyClientEncryption
+{
+    class Program

```

Comment:
Maybe this should be renamed to ClientEncryptionSample, I'm open to suggestions for this